.github/workflows/release.yml

```
@@ -0,0 +1,19 @@
+name: Release
+on:
+  push:
+    branches: [master, main]
```

```suggestion
    branches: [main]
```
no need for master, especially that someone could push it at some point by mistake.

---

Fixed